pquery-run.sh

```diff
@@ -853,7 +853,7 @@ pquery_test(){
     if [ "${VALGRIND_RUN}" == "0" ]; then
```

Comment:
have a new option. mysqld_kill_probability N. 
If it is set to 100, then do 100 Kill or MySQL fast shutdown.

It it is set to less than 100 then we do kill based on that probability. 
and rest we do normal shutdown

Reply:
This is a pquery script bug fix. The above request is not in the scope of this PR

Reply:
okay. but till now default was shutdown(whatever the reason). now you are setting the default to killing. 
but pquery 3 needs both of them. shutdown and kill. so rather replacing shutdown with the kill. 

I suggest pquery_script should have a new option to pick either.